### ISSUE TITLE
Switch from NetlifyCMS to StaticCMS

### ISSUE THREAD

**Issue by Marocco2** [delucis/astro-netlify-cms#61]
Hi!
Today StaticCMS has released its 1.0.0 version of the successor of NetlifyCMS, which it has been abandoned.
https://github.com/StaticJsCMS/static-cms/releases/tag/v1.0.0

It may be good switching ASAP to get better features and security in the future

**Comment by acharlop** [comment]
Hey I just put together a POC for working with StaticCMS based off of the work you did here. Please let me know if you are interested in collaborating or if I can publish `astro-static-cms` as a separate package. I see that you don't have a license file so either way I'd need your permission 🙏 
https://github.com/acharlop/astro-static-cms

**Comment by sean-au** [comment]
I'm super keen on moving forward with static cms... keen to hear where this is heading!

**Comment by onionhammer** [comment]
What about decap cms? Seems like the project might be revived

https://decapcms.org/docs/intro/

**Comment by sean-au** [comment]
Agreed decap has the backing and support needed for an open source project

**Comment by Marocco2** [comment]
> What about decap cms? Seems like the project might be revived
> 
> https://decapcms.org/docs/intro/

They haven't pushed a single commit other than rebranding ones. We can talk when they're going to publish a new release

**Comment by onionhammer** [comment]
@Marocco2 I dont think you shouldn't pursue astro support for StaticJSCms, but why not just have two astro plugins?

**Comment by Marocco2** [comment]
> @Marocco2 I dont think you shouldn't pursue astro support for StaticJSCms, but why not just have two astro plugins?

I didn't say that it shouldn't be considered _at all_. We must see what they're gonna do with NetlifyCMS repo before jumping up to the hype train. 
Still, everybody is free to do whatever they want.

**Comment by sean-au** [comment]
I vote close this thread as netlify CMS is transitioning to decap
